mllib/src/main/scala/org/apache/spark/ml/Predictor.scala

```diff
@@ -70,8 +68,8 @@ private[ml] trait PredictorParams extends Params
  * :: DeveloperApi ::
  * Abstraction for prediction problems (regression and classification). It accepts all NumericType
  * labels and will automatically cast it to DoubleType in [[fit()]]. If this predictor support
- * weights, it accepts all NumericType weights. And if the weightCol is available (defined and
- * non-empty), the weights will also be automatically cast to DoubleType in [[fit()]].
+ * weights, it accepts all NumericType weights, which will be automatically casted to DoubleType
```

Comment:
"If this predictor support" => "If this predictor supports"
"to DoubleType in in" => "to DoubleType in"